apps/editing-toolkit/editing-toolkit-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -68,7 +80,7 @@ function updateEditor() {
 			window.top.location.href = launchHref;
 		};
 
-		const handleLaunch = ( e: Event ) => {
+		const handleLaunch = async ( e: Event ) => {
```

Comment:
Is the `async` keyword needed?

Reply:
Nope, remove in https://github.com/Automattic/wp-calypso/pull/47514/commits/9f8c46ffbdda8f25abf2393938cdb437deeaf94a